server/actorController.js

```diff
@@ -5,8 +5,7 @@ const actorController = {
     // req.body.fullname should be an array with first and last name
     const { firstName, lastName } = req.body;
     // add actor to the database
-    // firstName = firstName.toLowercase();
-    const values = [firstName.toLowerCase(), lastName.toLowerCase()];
+    const values = [firstName.toLowerCase().trim(), lastName.toLowerCase().trim()];
```

Comment:
Actor names were put into the database as-is, without any validation or normalization.  
It was possible to submit an actor with a first name like `"Jeff "`.  
Trimming the names removes any extra spaces around the name, making the data a bit more normalized.